forms_pro/api/team.py

```
@@ -77,3 +89,131 @@ def switch_team(team_id: str) -> None:
+def remove_member_from_team(team_id: str, member_email: str) -> None:
+    """
+    Remove a member from a team
+    """
+
+    if not frappe.has_permission(
+        doctype="FP Team",
+        ptype="write",
+        doc=team_id,
+        user=frappe.session.user,
+    ):
+        raise frappe.PermissionError("You do not have permission to remove a member from this team")
```

Ruff (0.15.4)

[warning] Avoid specifying long messages outside the exception class

(TRY003)